src/main/java/com/commercetools/helper/mapper/impl/payment/BasePaymentMapperImpl.java

```diff
@@ -240,27 +255,22 @@ protected Details getTransactionDetails(@Nonnull CtpPaymentWithCart paymentWithC
     protected Stream<Item> mapCustomLineItemToPaypalPlusItem(@Nonnull CustomLineItem customLineItem, @Nonnull List<Locale> locales) {
         if (customLineItem.getDiscountedPricePerQuantity().size() > 0) {
             return customLineItem.getDiscountedPricePerQuantity().stream()
-                    .map(dlipfq -> createPaypalPlusItem(customLineItem.getName(), locales, dlipfq));
+                    .map(dlipfq -> createPaypalPlusItem(customLineItem.getName().get(locales), dlipfq));
```

Comment:
If you pass here not a concrete locale but a list of locales which locale will it take? Also, why do you pass a list here?

Reply:
It was the previous implementation to use a `List<Locale>`. In this case, the first matching locale will be used.

Reply:
How does it match exactly or it just takes the first available locale?

Reply:
>locales the locale which should be searched, the first exact match wins

This is from LocalizedString implementation. https://github.com/commercetools/commercetools-jvm-sdk/blob/cfa043b25ebec1656311643ca4201ec81a172fe2/commercetools-sdk-base/src/main/java/io/sphere/sdk/models/LocalizedString.java#L197

Reply:
We should take by default always a language defined as `languageCode` custom field on the payment (it is a required field for the integration: https://github.com/commercetools/commercetools-paypal-plus-integration) and only fall back to current behaviour if no values for `languageCode` found or `languageCode` is not defined.

Reply:
73ba3db1c7ac170d6850bf0a84e76576a6f116a5